README.md

```diff
@@ -147,9 +147,9 @@ end
 
 ### 'Handle' Actions
 
-Every action not excluded defines a default `handle_` variant receives
-pre-processed data and is expected to return an Ecto query or record. All of 
-the handle calls may also return a conn (including the result of a render 
+Every action not excluded defines a default `handle_` variant which receives
+pre-processed data and is expected to return an Ecto query or record. All of
+the handle calls may also return a conn (including the result of a render
```

Comment:
Had basically [subject] [verb] [object] [verb] [object] here.  If I understood your explanation in flowdock this makes it work grammatically and retains the meaning you intended.